src/client/QXmppRosterManager.h

```diff
@@ -119,6 +119,17 @@ public Q_SLOTS:
     /// signal will not be emitted.
     void subscriptionReceived(const QString &bareJid);
 
+    /// This signal is emitted when a contact asks to subscribe to your presence.
+    ///
+    /// You can either accept the request by calling acceptSubscription() or refuse it
+    /// by calling refuseSubscription().
+    ///
+    /// \note If you set QXmppConfiguration::autoAcceptSubscriptions() to true, this
+    /// signal will not be emitted.
+    ///
+    /// \since QXmpp 1.5
```

Comment:
Can you move this into the cpp file (like here: https://github.com/qxmpp-project/qxmpp/blob/master/src/client/QXmppAttentionManager.cpp#L58-L77)